File: TestApp/MovieApi/Models/Movie/GetTopRatedMovies/Request/GetTopRatedMoviesRequestModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MovieApi.Models.Movie.GetTopTatedMovies.Request
{
    public class TopRatedMoviesRequestModel
    {
        public string Language { get; set; }
        public int Page { get; set; }
        public string Region { get; set; }
    }
}

File: TestApp/WebApplication2/ClientApp/src/app/models/dataSupplier-models/GetTopRatedMovies/GetTopRatedMoviesRequest.model.ts
export interface IGetTopRatedMoviesRequest {
  language: string;
  page: number;
  region: string;
}

export class GetTopRatedMoviesRequestDto implements IGetTopRatedMoviesRequest {
  language: string;
  page: number;
  region: string;

  constructor(language: string, page: number, region?: string) {
    this.language = language;
    this.page = page;
    this.region = region;
  }
}
